content/graphql/basics/3-big-picture.md

```diff
@@ -63,8 +63,8 @@ As you learned in the previous chapter, the payload of a GraphQL query (or mutat
 
 When the server receives a query, it will call all the functions for the fields that are specified in the query's payload. It thus *resolves* the query and is able to retrieve the correct data for each field. Once all resolvers returned, the server will package data up in the format that was described by the query and send it back to the client.
 
-![](http://imgur.com/cP2i8Da.png)
-*Each field in the query corresponds to a [resolver function](http://graphql.org/learn/execution/#root-fields-resolvers). The GraphQL calls all required resolvers when a query comes in to fetch the specified data.*
+![GraphQL query and resolvers](http://imgur.com/cP2i8Da.png)
```

Comment:
The articles are not using currently the image description like this, could you remove it?